roles/cloud_connector/tasks/cleanup_receptor.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Find receptor configs
+  find:
+    paths: /etc/receptor
+    file_type: directory
+  register: receptor_configs
+
+- name: Identify Satellite ID from Satellite
+  ansible.builtin.uri:
+    url: "https://{{ foreman_cloud_connector_url }}/api/settings?search=name%20%3D%20instance_id"
```

Comment:
in the example playbook, `foreman_cloud_connector_url` contained the `https://` already (which makes it identical to `foreman_url` in FAM, which I like), so I guess it should be dropped here?